social-network/src/CreativityPage/Creativity.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import CreativityPage from './index';
+import ArticleItem from './ArticleItem';
+import ReactTestRenderer from 'react-test-renderer';
+
+describe('CreativityPage component', () => {
+    it('should render', () => {
+        const component = ReactTestRenderer.create(<CreativityPage />);
+        expect(component.toJSON()).toMatchSnapshot();
+    });
+});
+describe('ArticleItem component', () => {
+    it('should render', () => {
+        const component = ReactTestRenderer.create(<ArticleItem />);
```

Comment:
To get this test to pass, you'll want to pass in some fake data for the `articleMetaData` prop. Also, make sure to save the snapshot to git so that other developers can compare your test results.